calculator/src/App.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import Button from './components/Button';
+import Result from './components/Result';
+import './App.css';
+
+class App extends React.Component {
+  constructor(){
+    super();
+    this.state = { result: "" }
+  }
+
+  onClick = (symbol) => {
+    if(symbol === "="){
+      this.calculate()
+    } else if(symbol === "AC"){
+      this.reset()
+    } else if(symbol === "sign"){
+      this.signChange()
+    } else {
+      this.setState({ result: (this.state.result + symbol).toLocaleString("en") })
+    }
+  }
+
+  calculate = () => {
+    let resultCheck = ''
+    if(this.state.result.includes('--')){
+        resultCheck = this.state.result.replace('--','+')
+    }
+    else {
+        resultCheck = this.state.result
+    }
+    try {
+        this.setState({
+            // eslint-disable-next-line
+            result: (eval(resultCheck) || "" ) + ""
```

Comment:
using `eval()` is fine for this project, just an FYI is that using it is dangerous because having any string as an executable can make any website you use it on vulnerable. I frankly love `eval()` but it is what it is.....